static/app/components/feedback/widget/useFeedback.tsx

```diff
@@ -6,14 +6,17 @@ import ConfigStore from 'sentry/stores/configStore';
 import {useLegacyStore} from 'sentry/stores/useLegacyStore';
 import useAsyncSDKIntegrationStore from 'sentry/views/app/asyncSDKIntegrationProvider';
 
+type ArgumentTypes<F extends Function> = F extends (...args: infer A) => any ? A : never;
```

Comment:
Is this the same as [`Parameters`](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype)?

Reply:
You're totally right - thank you!